action/store/postgres.py

```diff
@@ -117,6 +117,13 @@ def _clean_column_default(value: str) -> Any:
             # Unwrap value cast to text as a string
             value = value[0 : -len("::text")]
             if value[0] == value[-1] == "'" and value.count("'") == 2:
-                value = value[1:-1]
-
-        return value
+                return value[1:-1]
+        elif value.isnumeric():
+            return int(value)
+        elif value.lower() in (
+            "true",
+            "false",
+        ):
+            return bool(value)
```

Comment:
Try `bool("FALSE")` in an interactive interpreter and prepare to be shocked

Reply:
Whoops 😅